src/initializers/migration-runner.php

```diff
@@ -99,28 +110,21 @@ public function run_migrations( $name, $migrations_table_name, $migrations_direc
 			return false;
 		}
 
+		$migrations = $this->loader->get_migrations( $name );
```

Comment:
`get_migrations` can return false. But here is no check for that. This will result in a warning  at the `array_keys` below and possibly try to run the migration on a `NULL` class.